cmd/snap/inhibit.go

```diff
@@ -25,12 +25,34 @@ import (
 	"os"
 	"time"
 
+	"github.com/godbus/dbus"
 	"github.com/snapcore/snapd/cmd/snaplock/runinhibit"
 	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/logger"
 	"github.com/snapcore/snapd/progress"
 	"github.com/snapcore/snapd/usersession/client"
 )
 
+func detectSnapDesktopIntegration(snapName string) bool {
```

Comment:
This function is not just detecting if the SnapDesktopIntegration is available, but it's actually setting the refresh status on it, so I think that the `detect` prefix is misleading here. Maybe call it `tryNotifySnapDesktopIntegration()`?

Reply:
Good point... I used tryNotifyRefreshToSnapDesktopIntegration(), because in the future we may want to notify more things.

Done.